Jenkinsfile

```diff
@@ -0,0 +1,184 @@
+/*
+ * This pipeline supports `minor`, `patch`, `snapshot` and `edge` releases. It also SHOULD
+ * support `major` releases, but it's untested.
+ *
+ * The Jenkins job should be configured with the following properties:
+ *
+ * - Disable concurrent builds
+ * - Parameters (all must be trimmed; all are strings):
+ *    - RELEASE_TYPE
+ *      defaultValue: minor
+ *      description: Valid values are: minor.
+ *   - OPERATOR_REPO
+ *      defaultValue: kiali/kiali-operator
+ *      description: The GitHub repo of the kiali-operator sources, in owner/repo format.
+ *   - OPERATOR_RELEASING_BRANCH
+ *      defaultValue: refs/heads/master
+ *      description: Branch of the kiali-operator repo to checkout and run the release
+ *   - QUAY_OPERATOR_NAME
+ *      defaultValue: quay.io/kiali/kiali-operator
+ *      description: The name of the Quay repository to push the operator release
+ */
+
+def bumpVersion(String versionType, String currentVersion) {
+  def split = currentVersion.split('\\.')
+    switch (versionType){
+      case "patch":
+        split[2]=1+Integer.parseInt(split[2])
+          break
```

Comment:
Why does the indentation here look different than the other `case` statements?

Reply:
It's VIM mixing tabs and white spaces when indenting. I'll fix.